layers/build-krd/dracut/soci/soci-settled.sh

```diff
@@ -163,7 +163,13 @@ soci_udev_settled() {
         chmod 700 /factory/secure
         cp /manifestCA.pem /factory/secure/
 
-        mkdir -p /config /scratch-writes /atomfs-store
+        # expand the pcr7data if needed
+        [ -e /pcr7data.cpio ] && [ ! -e /pcr7data ] && \
+            ( mkdir -p /pcr7data; cd /pcr7data; cpio -id < /pcr7data.cpio )
+        for d in config scratch-writes atomfs-store; do
```

Comment:
```suggestion
        # switch_root pivots into /sysroot and will delete anything on the initrd's rootfs; use a tmpfs mount to avoid
        for d in config scratch-writes atomfs-store; do
```